indico/modules/rb/client/js/modules/calendar/CalendarListView.jsx

```diff
@@ -35,13 +46,19 @@ class CalendarListView extends React.Component {
     roomFilters: PropTypes.object.isRequired,
     calendarFilters: PropTypes.object.isRequired,
     datePicker: PropTypes.object.isRequired,
+    linkData: PropTypes.object.isRequired,
```

Comment:
This triggers a PropTypes warning when viewing the booking list outside linking mode (e.g. when just going to `/rooms/calendar?view=list`). required values can't be null so you need to leave it optional and set null as the default value (super ugly, i know...)